packages/envoyproxy/data_stream/stats/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,239 @@
+---
+description: Pipeline for processing envoyproxy metrics
+processors:
+- set:
+    field: ecs.version
+    value: 8.11.0
+- set:
+    field: event.kind
+    value: metric
+- rename:
+    field: statsd
+    target_field: envoyproxy
+    ignore_missing: true
+- script:
+    lang: painless
+    description: This script processor iterates over the whole document to remove few fields.
+    source: |-
+      for (entry in ctx.envoyproxy.entrySet()) {
+          def k = entry.getKey();
+          def v = entry.getValue();
+
+          if (!(v instanceof String)) {
+              v.keySet().stream().filter(s -> s.startsWith('1m_rate') || s.startsWith('5m_rate') ||s.startsWith('15m_rate') ||
+              s.startsWith('p99_9') || s.startsWith('p75') || s.startsWith('p99') || s.startsWith('p95'))
+              .collect(Collectors.toList()).forEach(m -> v.remove(m))
+          }
+      }
+- dot_expander:
+    field: "*"
+    path: labels
+- rename:
+    field: labels.envoy.cluster_name
+    target_field: envoyproxy.cluster_name
+    ignore_missing: true
```

can we not just use single rename processor to just rename `labels.envoy` to `envoyproxy` here?
```suggestion
- rename:
    field: labels.envoy
    target_field: envoyproxy
    ignore_missing: true
```

---

Tried this and some events could not be indexed.